.github/workflows/pr-job-dispatcher.yml

```diff
@@ -0,0 +1,117 @@
+        run: |
+          is_docs_file() {
+            local f="$1"
+            [[ "$f" == doc/* ||
+               "$f" == admin/* ||
+               "$f" == src/sample.ceph.conf ||
+               "$f" == CodingStyle ||
+               "$f" == *.rst ||
+               "$f" == *.md ||
+               "$f" == COPYING* ||
+               "$f" == README.* ||
+               "$f" == SubmittingPatches ||
+               "$f" == .readthedocs.yml ||
+               "$f" == PendingReleaseNotes ]] && return 0
+            return 1
+          }
+      
+          only_docs=true
+
+          changed_files="${{ steps.get-pr-changed-files.outputs.all_changed_files }}"
+          for file in $changed_files; do
+            echo "Checking: [$file]"
+            if ! is_docs_file "$file"; then
+              echo "$file is not a docs file"
+              only_docs=false
+              break
+            fi
+          done
+
+          echo "only_docs=$only_docs" >> "$GITHUB_OUTPUT"
+
+      - name: Check if only container-related files changed
+        id: check-if-container-only-pr
+        run: |
+          is_container_file() {
+            local f="$1"
+            [[ "$f" == container/* ||
+               "$f" == Dockerfile.build ||
+               "$f" == src/script/buildcontainer-setup.sh ||
+               "$f" == src/script/build-with-container.py ]] && return 0
+            return 1
+          }
+      
+          only_container=true
+
+          changed_files="${{ steps.get-pr-changed-files.outputs.all_changed_files }}"
+          for file in $changed_files; do
+            echo "Checking: [$file]"
+            if ! is_container_file "$file"; then
+              echo "$file is not a container file"
+              only_container=false
+              break
+            fi
+          done
+
+          echo "only_container=$only_container" >> "$GITHUB_OUTPUT"
+
+  trigger-fake-status-checks:
+    if: needs.pr-job-dispatcher.outputs.only_docs == 'true' || needs.pr-job-dispatcher.outputs.only_container == 'true'
+    permissions:
+      actions: write
+    needs: pr-job-dispatcher
+    uses: ./.github/workflows/pr-checks-faker.yml
+    with:
+      pr_number: ${{ needs.pr-job-dispatcher.outputs.pr_number }}
+      pr_sha: ${{ needs.pr-job-dispatcher.outputs.pr_sha }}
+      triggered_by: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+
+  trigger-jenkins-jobs:
+    if: needs.pr-job-dispatcher.outputs.only_docs != 'true' || needs.pr-job-dispatcher.outputs.only_container != 'true'
+    permissions:
+    needs: pr-job-dispatcher
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Jenkins jobs with parameters
+        run: |
+          GH_PULL_REQUEST_ID="${{ needs.pr-job-dispatcher.outputs.pr_number }}"
+          GH_PULL_REQUEST_SHA="${{ needs.pr-job-dispatcher.outputs.pr_sha }}"
+
+          for job in ceph-pull-requests; do
+            curl -X POST "https://jenkins.ceph.com/job/$job/buildWithParameters" \
```

Comment:
it occurs to me now: any kind of retry or timeout we want for the curls (just to make them bulletproof)?